schemas/context/experienceevent-privacy.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+   "meta:license":[
+      "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.adobe.com/xdm/context/experienceevent-privacy",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Profile privacy",
```

Comment:
Please change this to Experience Event Privacy.

Reply:
fixed.